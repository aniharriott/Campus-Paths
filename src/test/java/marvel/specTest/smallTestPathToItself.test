# Find a path to itself using smallTest.

LoadGraph small smallTest.tsv

FindPath small a a