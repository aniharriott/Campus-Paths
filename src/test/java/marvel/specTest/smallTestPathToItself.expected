# Find a path to itself using smallTest.

loaded graph small

path from a to a: